notice_and_comment/settings/base.py

```diff
@@ -1,6 +1,9 @@
 import json
 import os
 
+from cfenv import AppEnv
+env = AppEnv()
+
```

Comment:
Sorry to confuse. I think you only want to use cfenv in the production version; we don't need to worry about cloud.gov locally.